test/extend.js

```diff
@@ -114,6 +114,25 @@ describe('Extend', () => {
     });
   });
 
+  it('should extend layout with data', () => {
+    mfs.writeFileSync('./layout.html', `
+            <head><block name="head">head</block></head>
+            <body class="{{ bodyclass }}"><block name="body">body</block></body>
+        `);
+
+    return init(
+      `<extends src="layout.html" data='{"bodyclass": "home"}'>
```

Comment:
in the documentation you specified the `locals` in the tests you are using `date`. It may be worth using it everywhere `locals` ? 

I think it would be better to have a single agreement with `posthtml-expressions`

Reply:
Yeah, you're probably right ^^ Working on it